pom.xml

```diff
@@ -201,7 +201,9 @@
                                 <keycloak.image>quay.io/keycloak/keycloak:15.0.1</keycloak.image>
                                 <rhsso.73.image>registry.access.redhat.com/redhat-sso-7/sso73-openshift</rhsso.73.image>
                                 <postgresql.13.image>quay.io/bitnami/postgresql:13.5.0</postgresql.13.image>
+                                <elastic.71.image>quay.io/bitnami/elasticsearch:7.10.2</elastic.71.image>
                                 <mysql.57.image>quay.io/bitnami/mysql:5.7.32</mysql.57.image>
+                                <mysql.80c.image>quay.io/bitnami/mysql:8.0</mysql.80c.image>
```

Comment:
this is the community version of Mysql 8.0 used in baremetal

Reply:
Maybe `mysql.bitnami.80.image`? The 'c' in the name is not very clear. :)